submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/data/SubmissionRequestDocDataRepository.kt

```diff
@@ -254,6 +260,16 @@ class SubmissionRequestDocDataRepository(
         mongoTemplate.updateFirst(query, update, DocSubmissionRequest::class.java).awaitSingleOrNull()
     }
 
+    suspend fun increaseIndex(
```

Comment:
rather than two functions we can have this one and in the other one call this with 1